Stripe/STPPaymentMethodEnums.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  STPPaymentMethodEnums.swift
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/12/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The type of the PaymentMethod.
+@objc public enum STPPaymentMethodType: Int {
+  /// A card payment method.
+  case card
+  /// An Alipay payment method.
+  case alipay
+  /// A GrabPay payment method.
+  case grabPay
+  /// An iDEAL payment method.
+  @objc(STPPaymentMethodTypeiDEAL) case iDEAL
+  /// An FPX payment method.
+  case FPX
```

Comment:
does this need an explicit `@objc(` name?

Reply:
Yeah, it'll bridge by automatically capitalizing the first letter (e.g. "STPPaymentMethodTypeIDEAL").